tools/cli/domain_migration_command.go

```diff
@@ -61,7 +61,7 @@ var (
       Name: {{.DomainInfo.Name}}
       UUID: {{.DomainInfo.UUID}}
   {{- end}}
-  {{- if ne (len .MismatchedDomainMetaData) 0 }}   
+  {{- if ne (len .MismatchedDomainMetaData) 0 }}
     Mismatched Domain Meta Data: {{.MismatchedDomainMetaData}}
   {{- end }}
```

Comment:
yea, three trailing spaces kinda makes no sense.  if it was 4 I might wonder if it's a meaningful blank indented line, but that doesn't seem like the case.